utils/common/atomicops.h

```diff
@@ -165,6 +165,28 @@ inline bool atomicCAS(volatile T* mem, T comp, T swap)
 #endif
 }
 
+// implements a zero out of a variable
+template <typename T>
+inline void atomicZero(volatile T* mem)
+{
+#ifdef _MSC_VER
```

Comment:
I doubt we will have MCS on Windows again.

Reply:
Nevertheless, it doesn't hurt to keep the easy stuff compatible. You never know.